routes/artifacts.js

```diff
@@ -18,7 +18,7 @@ api.declare({
       'assume:worker-id:<workerGroup>/<workerId>'
     ], [
       'queue:create-artifact:<name>',
-      'queue:claim-task:<taskId>/<runId>'
+      'queue:claim-run-id:<taskId>/<runId>'
```

Comment:
@petemoore, okay can we agree this one isn't necessary... as we can just not give `queue:create-artifact:` ?